### PR TITLE
Updated sig-apps group and slack

### PR DESCRIPTION
**Pull request by mattfarina**

### CODE REVIEW

**Comment by mattfarina** [comment]
@sarahnovotny note, I still don't have write access to merge my own or others for sig-apps. That's why I'm pinging you.

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Much of the community activity is organized into a community meeting and numerou
 | Name | Leads | Group | Slack Channel | Meetings |
 |------|-------|-------|---------------|----------|
 | [API Machinery](sig-api-machinery/README.md) | [@lavalamp (Daniel Smith, Google)](https://github.com/lavalamp) | [Group](https://groups.google.com/forum/#!forum/kubernetes-sig-api-machinery) | [#sig-api-machinery](https://kubernetes.slack.com/messages/sig-api-machinery/) | Every other Wednesday at 11:00 AM PST |
-| [Apps](sig-apps/README.md) | [@michelleN (Michelle Noorali, Deis)](https://github.com/michelleN)<br>[@mattfarina (Matt Farina, HPE)](https://github.com/mattfarina) | Group TBD | Slack TBD | Weekly on Wednesday at 9:00 AM PST |
+| [Apps](sig-apps/README.md) | [@michelleN (Michelle Noorali, Deis)](https://github.com/michelleN)<br>[@mattfarina (Matt Farina, HPE)](https://github.com/mattfarina) | [Group](https://groups.google.com/forum/#!forum/kubernetes-sig-apps) | [#sig-apps](https://kubernetes.slack.com/messages/sig-auth) | Weekly on Wednesday at 9:00 AM PST |
```
Comment (michelleN) [inline README.md]: shouldn't it be "sig-apps" instead of "sig-auth"❓

**Comment by sarahnovotny** [comment]
I don't have permission to add @mattfarina or @michelleN to the kubernetes organization (and by extension the sig-apps group.)  could someone from @kubernetes/kubernetes-admin give me that permission?  We also need anyone in the org to be able to commit to the kubernetes/community repo.  thanks!

**Comment by bgrant0607** [comment]
@sarahnovotny Done

**Comment by bgrant0607** [comment]
@mattfarina @michelleN You need to accept the invitations to join the kubernetes org.

**Comment by sarahnovotny** [comment]
thanks, @bgrant0607 .

**Comment by davidopp** [comment]
sig-apps does not appear in the sidebar on
https://github.com/kubernetes/kubernetes/wiki
can someone add it?

Or even better, maybe we should just remove all of the SIGs from the sidebar (there are so many now, that it has become pretty cluttered), since there is already a link to 
https://github.com/kubernetes/kubernetes/wiki/Special-Interest-Groups-(SIGs)
there.

**Comment by davidopp** [comment]
Oh, actually, it seems that we should
(1) delete all of the specific SIGs in the sidebar
(2) make the "Special Interest Groups SIGs)" link to 
https://github.com/kubernetes/community/README.md
instead of to
https://github.com/kubernetes/kubernetes/wiki/Special-Interest-Groups-(SIGs)

???

**Comment by bgrant0607** [comment]
I'd like all the SIG content to move to this repo.

The existing overall SIG wiki doesn't list all the SIGs, and needs to be updated

**Pull request merged**